packages/core/src/sims/ranged/mch/mch_sheet_sim.ts

```diff
@@ -0,0 +1,512 @@
+        // Pile Bunker
+        // Crowned Collider
+        const automatonQueenDelay = 5.6;
+        const automatonQueenDelayBetweenAbilities = 1.5;
+        cp.AutomatonQueenAbilityUsages.push({
+            // Arm Punch 1
+            ability: {
+                ...Actions.AutomatonQueenArmPunch,
+                potency: 2.4 * cp.gauge.batteryGauge},
+            // potency per 1 battery
+            usageTime: cp.currentTime + automatonQueenDelay,
+        });
+        cp.AutomatonQueenAbilityUsages.push({
+            // Arm Punch 2
+            ability: {
+                ...Actions.AutomatonQueenArmPunch,
+                potency: 2.4 * cp.gauge.batteryGauge},
+            // potency per 1 battery
+            usageTime: cp.currentTime + automatonQueenDelay + 1 * automatonQueenDelayBetweenAbilities,
+        });
+        cp.AutomatonQueenAbilityUsages.push({
+            // Arm Punch 3
+            ability: {
+                ...Actions.AutomatonQueenArmPunch,
+                potency: 2.4 * cp.gauge.batteryGauge},
+            // potency per 1 battery
+            usageTime: cp.currentTime + automatonQueenDelay + 2 * automatonQueenDelayBetweenAbilities,
+        });
+        cp.AutomatonQueenAbilityUsages.push({
+            // Arm Punch 4
+            ability: {
+                ...Actions.AutomatonQueenArmPunch,
+                potency: 2.4 * cp.gauge.batteryGauge},
+            // potency per 1 battery
+            usageTime: cp.currentTime + automatonQueenDelay + 3 * automatonQueenDelayBetweenAbilities,
+        });
+        cp.AutomatonQueenAbilityUsages.push({
+            // Arm Punch 5
+            ability: {
+                ...Actions.AutomatonQueenArmPunch,
+                potency: 2.4 * cp.gauge.batteryGauge},
+            // potency per 1 battery
+            usageTime: cp.currentTime + automatonQueenDelay + 4 * automatonQueenDelayBetweenAbilities,
+        });
+        cp.AutomatonQueenAbilityUsages.push({
+            // Pile Bunker
+            ability: {
+                ...Actions.AutomatonQueenArmPunch,
+                potency: 6.8 * cp.gauge.batteryGauge},
+            // potency per 1 battery
+            usageTime: cp.currentTime + automatonQueenDelay + 5 * automatonQueenDelayBetweenAbilities,
+        });
+        cp.AutomatonQueenAbilityUsages.push({
+            // Crowned Collider
+            ability: {
+                ...Actions.AutomatonQueenArmPunch,
+                potency: 7.8 * cp.gauge.batteryGauge},
+            // potency per 1 battery
+            usageTime: cp.currentTime + automatonQueenDelay + 6 * automatonQueenDelayBetweenAbilities + 1,
+            // crowned collider has a recast time of 2.5s instead of 1.5s
+        });
+        cp.gauge.batteryGauge = 0;
+    }
+
+    use(cp: MchCycleProcessor, ability: Ability): AbilityUseResult {
+        if (cp.currentTime >= cp.totalTime) {
+            return null;
+        }
+
+        let mchAbility = ability as MchAbility;

```

Comment:
```suggestion
        const mchAbility = ability as MchAbility;
```
fixes the lint error